src/richie/apps/courses/cms_plugins.py

```diff
@@ -149,3 +150,25 @@ def render(self, context, instance, placeholder):
             }
         )
         return context
+
+
+@plugin_pool.register_plugin
+class ProgramPlugin(CMSPluginBase):
+    """
+    Program plugin displays a group of course selections
```

Comment:
not really :wink: it displays the program's cover image and title...
```suggestion
    Program plugin displays a course's information on other pages
```